pkg/printers/template.go

```diff
@@ -33,7 +34,10 @@ type TemplatePrinter struct {
 
 func NewTemplatePrinter(tmpl []byte) (*TemplatePrinter, error) {
 	t, err := template.New("output").
-		Funcs(template.FuncMap{"exists": exists}).
+		Funcs(template.FuncMap{
+			"exists": exists,
+			"b64dec": base64decode,
```

Comment:
Is b64dec a name we're happy with long term? I have a slight preference for being explicit, e.g. base64decode

Reply:
Also, the fact that that's the name you picked for the actual function (which I hadn't noticed) makes me smile

Reply:
:) Thanks @liggitt ! I picked `b64dec` for compatibility with [sprig](https://github.com/Masterminds/sprig) on the off chance that folks might want to expand the set of functions using that library.

Reply:
(and I got used to sprig from using Helm, so thought it would be good to be consistent.)

If you'd prefer `base64decode` I'm not stuck on `b64dec`, just wanted to explain where the name came from.

Reply:
I'd prefer `base64decode` too.

Reply:
OK. Sorry to be slightly stuck on this, but would it be a reasonable compromise to support both `base64decode` and `b64dec` to reduce frustration for folks who are also writing helm templates? I say this because I got here by writing a helm template that needed to print out some initial values for secret data for the user doing the install, and I'm sure that me three weeks from now will trip over the different name.

Feel free to say no; I realize that having multiple names for the same function is potentially confusing in a different direction.